src/EnergyPlus/DataSystemVariables.cc

```diff
@@ -95,45 +95,41 @@ namespace DataSystemVariables {
     // Thus, all variables in this module must be PUBLIC.
 
     // MODULE PARAMETER DEFINITIONS:
-    int const iASCII_CR(13);   // endline value when just CR instead of CR/LF
-    int const iUnicode_end(0); // endline value when Unicode file
-    char const tabchar('\t');
-
-    std::string const DDOnlyEnvVar("DDONLY");       // Only run design days
-    std::string const ReverseDDEnvVar("REVERSEDD"); // Reverse DD during run
-    std::string const DisableGLHECachingEnvVar("DISABLEGLHECACHING");
-    std::string const FullAnnualSimulation("FULLANNUALRUN"); // Generate annual run
-    std::string const cDeveloperFlag("DeveloperFlag");
-    std::string const cDisplayAllWarnings("DisplayAllWarnings");
-    std::string const cDisplayExtraWarnings("DisplayExtraWarnings");
-    std::string const cDisplayAdvancedReportVariables("DisplayAdvancedReportVariables");
-    std::string const cDisplayUnusedObjects("DisplayUnusedObjects");
-    std::string const cDisplayUnusedSchedules("DisplayUnusedSchedules");
-    std::string const cDisplayZoneAirHeatBalanceOffBalance("DisplayZoneAirHeatBalanceOffBalance");
-    std::string const cSortIDD("SortIDD");
-    std::string const cReportDuringWarmup("ReportDuringWarmup");
-    std::string const cReportDuringHVACSizingSimulation("REPORTDURINGHVACSIZINGSIMULATION");
-    std::string const cIgnoreSolarRadiation("IgnoreSolarRadiation");
-    std::string const cIgnoreBeamRadiation("IgnoreBeamRadiation");
-    std::string const cIgnoreDiffuseRadiation("IgnoreDiffuseRadiation");
-    std::string const cSutherlandHodgman("SutherlandHodgman");
-    std::string const cSlaterBarsky("SlaterBarsky");
-    std::string const cMinimalSurfaceVariables("CreateMinimalSurfaceVariables");
-    std::string const cMinimalShadowing("MinimalShadowing");
-    std::string const cNumActiveSims("cntActv");
-    std::string const cInputPath1("epin");       // EP-Launch setting.  Full path + project name
-    std::string const cInputPath2("input_path"); // RunEplus.bat setting.  Full path
-    std::string const cProgramPath("program_path");
-    std::string const cTimingFlag("TimingFlag");
-    std::string const TrackAirLoopEnvVar("TRACK_AIRLOOP"); // To generate a file with runtime statistics
+
+    constexpr const char *DDOnlyEnvVar("DDONLY");       // Only run design days
```

Comment:
I do wonder how many (if any) of these are needed outside of the worker function(s) below.  That's for another day, now that these are constexpr and not a problem anymore.